_posts/2018-04-27-scrum-vs-kanban.md

```diff
@@ -0,0 +1,131 @@
+It will disrupt the team's flow and velocity of the team will go down. 
+Once you’ve chosen a Sprint Length, your team should stick to it.
+
+#### Not having the same Sprint rhythm as another team
+In larger projects, teams will be dependent on one another. 
+Not having the same Sprint rhythm will cause issues because you’ll have to wait for the other team to be ready for you, or vice versa, which would once again block progress.
+
+#### The Scrum Master being too involved
+A Scrum master is a dedicated role in the team.
+It is very hard to be a Scrum master and develop at the same time. 
+He or she should be like a firefighter, be idle at times, but be ready when an emergency occurs. 
+
+#### Trying to solve problems within a Daily Standup
+The Standup has to be a short time boxed meeting. Answering the questions and going forward. 
+If there are problems, they should be handled before or after the Standup, not during. 
+
+## Kanban
+
+Kanban is a continuous process. 
+There isn’t a Sprint backlog. Only the columns on the board and the prioritized product backlog. 
+
+The Kanban Board is crucial within Kanban. 
+It is usually split intp 3 columns: To Do, Doing and Done. 
+More columns can be added to fit your needs but those 3 have to be there. 
+
+To limit the work that can be done there is a Work in Progress limit on every column on the Kanban board which is related to the team’s capacity. 
+For example, a team of 4 developers would be able to handle 6 to 8 stories at the time. 
+The lower the limit the faster work will pass through the board. 
+Having it too low though will result in a bottleneck situation. 
+
+Kanban has a Daily Standup meeting, a 15-minute standup meeting in which every member of the development team answers 3 simple questions: 
+What have I done yesterday, what will I do today, are there any problems that block me in going forward with my task. 
+
+Demo meetings are held at regular intervals.
+During these meetings all stakeholders are invited to a demonstration of the new functionality since the last demo. 
+
+Kanban also has retrospective meetings, which have the goal to improve the way of working of the team and solve any issues that have risen since the last retrospective. 
+
+In Kanban there are multiple ways you can do retrospectives.
+One way would be to do them on a regular basis (bit Scrum like). 
+Although this doesn’t fit in the Kanban way of working,
+this provides a welcome disruption of the daily workflow and if conducted well the retrospective will result in a morale boost for the team since they have been listened to and know problems they have will be handled or have been handled during the retro.
+
+Since having retrospectives on a regular basis kind of goes against the way of working in Kanban, a simpler way of holding retrospective has been popping up the ‘Stop and Solve‘. 
+This means that every time there is an issue (that the person who encountered it can’t solve it on their own), 
```

Comment:
that the person who encountered it can't solve on their own (without the "it" after "solve")